compiler/test/src/org/jetbrains/dukat/compiler/tests/httpService/CliHttpService.kt

```diff
@@ -4,7 +4,7 @@ import kotlinx.serialization.UnstableDefault
 import org.jetbrains.dukat.compiler.tests.NodeResolver
 import org.jetbrains.dukat.compiler.tests.core.TestConfig
 
-class CliHttpService(
+class CliHttpService @OptIn(UnstableDefault::class) constructor(
```

Comment:
this looks like something that has nothing to do with the goal of this PR - the thing is that CliHttpService is under heavy refactoring right now so I'd rather just won't include this one at all

Reply:
Unfortunately I can't build without this -- gives me warning and we have werror. I'll try to disable this warning though